recipes/eastl/all/conanfile.py

```diff
@@ -36,7 +36,7 @@ def configure(self):
 
         if (self.settings.compiler == "gcc" and Version(self.settings.compiler.version.value) < "5") or \
                 (self.settings.compiler == "clang" and Version(self.settings.compiler.version.value) < "3.4") or \
-                (self.settings.compiler == "Visual Studio" and Version(self.settings.compiler.version.value) < "2015"):
+                (self.settings.compiler == "Visual Studio" and Version(self.settings.compiler.version.value) < "14"):
```

Comment:
you can use `Version(self.settings.compiler.version)`, but it's okay.

Reply:
fixed